.brigade/brigade.ts

```diff
@@ -337,30 +338,11 @@ events.on("brigade.sh/github", "ci:pipeline_requested", async event => {
       buildObserverJob(event),
       buildSchedulerJob(event),
       buildWorkerJob(event),
-      buildBrigadierJob(event),  
+      buildBrigadierJob(event),
       buildCLIJob(event)
     ),
     testIntegrationJob(event)
   ).run()
-  if (event.worker?.git?.ref == "main") {
-    // Push "edge" images.
-    //
-    // npm packages MUST be semantically versioned, so we DON'T publish an
-    // edge brigadier package.
-    //
-    // To keep our github releases page tidy, we're also not publishing "edge"
-    // CLI binaries.
-    await new ConcurrentGroup(
-      pushArtemisJob(event),
-      pushAPIServerJob(event),
-      pushGitInitializerJob(event),
-      pushLoggerLinuxJob(event),
-      pushObserverJob(event),
-      pushSchedulerJob(event),
-      pushWorkerJob(event),
-      publishBrigadierDocsJob(event),
-    ).run()
-  }
```

Comment:
We used to publish "edge" images on merge to master. This has really been making a big mess of our public Docker Hub repos, so we stop doing it here.